src/Http/TracedResponse.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the worldia/instrumentation-bundle package.
+ * (c) Worldia <developers@worldia.com>
+ */
+
+namespace Instrumentation\Http;
+
+use OpenTelemetry\API\Trace\SpanInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+class TracedResponse implements ResponseInterface
```

Comment:
Should implement `Symfony\Component\HttpClient\Response\StreamableInterface`, ref. https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/HttpClient/Response/TraceableResponse.php